website/scripts/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+MMVERSION=${1-0.3.35}
```

Comment:
Is there a reason to use this formatting instead of the more universal syntax of `MMVERSION="0.3.35"`? I'm not a bash expert, so maybe there's a reason to use 1- formatting instead?

Reply:
well, it was intended to be passed in from the calling Makefile (which also contains this variable), but I apparently forgot to pass it 🙃 so that it is falling back on the default here.

i'll tweak that.